doc/generic/pgf/text-en/pgfmanual-en-pgfkeys.tex

```diff
@@ -1253,6 +1253,22 @@ \subsubsection{Defining Value-, Macro-, If- and Choice-Keys}
     and, thus, |\pgfkeysnovalue| will be stored in |/my key|.
 
     To retrieve the value stored in a key, the handler |/.get| is used.
+    
+    \medskip
+    \emph{Remark:} A key can both store a value and execute commands. If so,
+    using \meta{key} will \emph{always} execute commands with the passed
+    value, or the default value |\pgfkeysnovalue| if no value is provided.
+    Note that the stored value is never used. To update the stored value, the
+    handler |/.initial| or command |\pgfkeyssetvalue| is used.
```

Comment:
Please add that this will not be changed for compatibility.  Because you did fix it at some point but that broke other uses and we had to roll back.

Reply:
Added a footnote.